lib/services.template.ejs

```diff
@@ -267,7 +267,9 @@ module
       props.forEach(function(name) {
         self[name] = load(name);
       });
-      this.rememberMe = undefined;
+      if ( !this.rememberMe ){
```

Comment:
BTW I am still not convinced we need to unset rememberMe here, see https://github.com/strongloop/loopback-sdk-angular/pull/193#discussion_r45737168 What are your thoughts?